meson.build

```diff
@@ -7,11 +7,14 @@ project('vala-language-server', 'vala', 'c',
 )
 
 valac = meson.get_compiler('vala')
-# libvala_version = run_command(valac, '--api-version').stdout().strip()
-# 
-# libvala = dependency('libvala-@0@'.format(libvala_version))
+libvala_version = run_command(valac, '--api-version').stdout().strip()
+message(libvala_version)
```

Comment:
Do you think you could change this to something like `message('libvala ' + libvala_version)` so that the message is more informative?